src/components/anchor-navigation/anchor-navigation-item/anchor-navigation-item.style.ts

```diff
@@ -31,6 +31,8 @@ const StyledNavigationItem = styled.li<StyledNavigationItemProps>`
       css`
         background-color: var(--colorsActionMajorYang100);
         border-left-color: var(--colorsActionMajor500);
+        border-top-right-radius: var(--borderRadius100);
```

Comment:
Looks like you'll need to add some styling for the hover state as well

![image](https://user-images.githubusercontent.com/44157880/216064467-fb151d9e-5771-46e4-ab36-b97c6a36e8e4.png)